src/entity-manager.js

```diff
@@ -150,6 +151,14 @@ export class EntityManager {
       resource = entity;
     }
 
+     // Set the validator.
+    if (instance.hasValidation() && !(instance.getValidator())) {
+      let validator = this.container.invoke(StandardValidator);
+      this.container.registerInstance(StandardValidator, validator);
```

Comment:
fyi: that makes the validator transient.

 whether that's needed is another question